sysdump/sysdump.go

```diff
@@ -291,13 +294,30 @@ func NewCollector(k KubernetesClient, o Options, startTime time.Time, cliVersion
 	return &c, nil
 }
 
-func (c *Collector) GatherResourceUnstructured(ctx context.Context, r schema.GroupVersionResource, fname string) error {
+func (c *Collector) GatherResourceUnstructured(ctx context.Context, r schema.GroupVersionResource, fname string, keep ...string) error {
```

Comment:
Could we please document `keep ...string`? I can't guess what kind of parameter needs can be provided without looking at the method implementation.